api/misc.go

```diff
@@ -1,13 +1,13 @@
 package api
 
 import (
-	"io"
-	"net/http"
-	"strconv"
-
 	"github.com/pkg/errors"
 	"github.com/rancher/go-rancher/api"
 	"github.com/rancher/go-rancher/client"
+	"github.com/rancher/longhorn-manager/manager"
+	"io"
```

Comment:
Don't change the order of import. It's ordered. The first part is go native ones, then outside importer, then local ones.

So it should be:
```
"io"
...

"github.com/xxx"
...

"github.com/rancher/longhorn-manager/manager"

```